testing/integration/logs_ingestion_test.go

```diff
@@ -281,7 +281,7 @@ func testMonitoringLogsAreShipped(
 			"add_cloud_metadata: received error failed with http status code 404",                                                                                                                             // okay for the cloud metadata to not work
 			"add_cloud_metadata: received error failed fetching EC2 Identity Document: not found, Signing",                                                                                                    // okay for the cloud metadata to not work
 			"add_cloud_metadata: received error failed fetching EC2 Identity Document: operation error ec2imds: GetInstanceIdentityDocument, http response error StatusCode: 404, request to EC2 IMDS failed", // okay for the cloud metadata to not work
-			"failed to invoke rollback watcher: failed to start Upgrade Watcher: fork/exec /var/lib/elastic-agent/elastic-agent: no such file or directory",                                                   // on debian this happens probably need to fix.
+			"SDK 2024/05/15 17:49:30 WARN falling back to IMDSv1: operation error ec2imds: getToken, http response error StatusCode: 405, request to EC2 IMDS failed",                                         // okay for the cloud metadata to not work
```

Comment:
How is that going to work, it has a timestamp in it?

Reply:
This is a good point, I just blindly copy pasted this without looking too closely. It's going to need to be a pattern, or I need to go remove this log message from Beats.

Reply:
What's worse is this isn't even a JSON log, it is just getting dumped into stderr and we are picking it up. I need to figure out where this one comes from.

Reply:
I am going to look at just fixing the logging in add_cloud_metadata in general. Users trying to build a similar alert run into this exact same problem.